### PR TITLE
Fix wider with to LHS of UI. More space for source preview.

### LINKED ISSUE #869: UI Polish: Widen Source List
## Problem
Not sure why (cuz I know Allie worked very hard to get this right!), but the Source List is much narrower than it should be, and the Conversation Pane far wider than it should be.

Screenshot below, shows the difference between the grid used in my mockups (top), and the UI the Client loads with (bottom). 

image

## Solution
Developer fanciness happening?
When the list is wider, more preview text can be visible to Journalists, which will help them in their work, more. The Conversation Pane just doesn't need to be so wide, too.

### PR DESCRIPTION
**Pull request by ntoll**
# Description

Fixes #869.

TL;DR width of the LHS pane, width of the preview in the source preview widget, textwrap increased to 80 [was 70] characters a line. Screenie:

wider_left

# Test Plan

@ninavizz looks at it and provides comments. :eyes:

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:

 - [ ] I have tested these changes in the appropriate Qubes environment
 - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
 - [X] These changes should not need testing in Qubes

If these changes add or remove files other than client code, packaging logic (e.g., the AppArmor profile) may need to be updated. Please check as applicable:

 - [ ] I have submitted a separate PR to the [packaging repo](https://github.com/freedomofpress/securedrop-debian-packaging)
 - [X] No update to the packaging logic (e.g., AppArmor profile) is required for these changes
 - [ ] I don't know and would appreciate guidance

### CODE REVIEW

**Comment by ninavizz** [comment]
Hi Nicholas! This is a great start, and mostly great in general! If we could address two things, I'll be very happs to sign off. The width change in the Source List (or LHS as I think you're referring to it?) is great and can remain. TY!! :)
- [ ] Narrow the width of the preview text's text-block by the same width as one of those "the" entries from the top Source
- [ ] How can we fill-up more of that 2nd line with more preview text? Can we restrict whatever is visible in the first 2 lines of text in the widget, or do we have to do character-count? IF the latter, then mebbe increase the textwrap to 100 (if you only increased it from 70 to 80, before?). 

I'd also love to see a GIF of how the panes all dynamically re-size as the Client window re-sizes, now, too. Danke!! :)

**Comment by creviera** [comment]
It looks like the preview text is still wrapping in an unexpected way, even after you made it 10 pixels wider, see how a newline is added after "onion":

Screenshot from 2020-03-10 14-17-35

**Comment by creviera** [comment]
Also long strings like that long filename A_B_C_... gets cut off instead of wrapped. We have an issue for this one: https://github.com/freedomofpress/securedrop-client/issues/877.

**Comment by ntoll** [comment]
OK... so I've spent quite some time on this. Put simply, because we don't use a fixed width font then we get the problems @creviera describes since the line length in characters doesn't make a uniform width in terms of pixels used.

Given the scope of this PR is the new width of the column and the word wrapping bug is reported in #877 I believe we should merge this and then fix the word wrapping via a PR to address #877.

Make sense..?

**Comment by ntoll** [comment]
So #926 should fix the word wrap issue related to the behaviour observed during the testing of this PR.
